app/errors.py

```diff
@@ -0,0 +1,14 @@
+class VaccineError(Exception):
+    pass
+
+
+class NotVaccinatedError(VaccineError):
+    pass
```

Comment:
Consider adding a descriptive error message to the `NotVaccinatedError` exception. You can add a default message in the `__init__` method of the exception class.